packages/frontend/src/scripts/index.ts

```diff
@@ -33,6 +34,7 @@ initStateUpdateStats()
 // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
 if (USE_NEW_DESIGN) {
   initForcedActionForm()
+  initSpotWithdrawalForm()
```

Comment:
If that's what I think it is then shouldn't it be name `initSpotForcedWithdrawalForm`?

Reply:
That's right